### DISCUSSION TITLE
Off-Topic

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#985] [General]
![image](https://user-images.githubusercontent.com/1283445/213860681-82fac54b-868a-4c56-87cf-52918388ef1e.png)

**Comment by wolfgang-ch** [comment]
Now with a new cap and with a scarf

![2023-01-22___DSC02716](https://user-images.githubusercontent.com/1283445/214492598-669f9f92-6d9d-42dc-af61-785c5dd96902.jpg)

**Comment by FJBDev** [comment]
A bit more here 😄 

![image](https://user-images.githubusercontent.com/25235665/214584438-6a0cd31f-fb39-4242-9326-64d6cf18de0a.png)

**Reply by wolfgang-ch** [reply]
2000/2001 I lived in [Acton](https://www.acton-ma.gov/) for 11 months and in this winter there was also a lot of snow. 

I worked there to build together with my colleage [Thomas Gumz](https://twitter.com/thomasgumz) the Domino Webadmin 5 which is now owned by HCL https://help.hcltechsw.com/domino/10.0.1/admn_thewebadministrator_c.html

![image](https://user-images.githubusercontent.com/1283445/214763491-8d85cf5b-c3f7-471d-b116-9c09b7541d84.png)

2016 I visited this location again but first I didn't saw it because now it looks differently

![image](https://user-images.githubusercontent.com/1283445/214764147-61785d3a-5ce3-43ed-b5be-d43757d756a0.png)

**Comment by telemaxx** [comment]
![20230122_125248](https://user-images.githubusercontent.com/22706487/214619915-58506290-ed39-4403-bcc9-6c62f9a72933.jpg)

here we had also some snow last weekend.
